### PR TITLE
Update CSV merger to inject Volume Replication Operator CRDs

### PR DESCRIPTION
**Pull request by umangachapagain**
Volume Replication Operator CRDs are not owned by default.
Since it is the part of ocs-operator bundle, we manually
create a mock CSV for volume replication operator and allow
ocs-operator to assume ownership of it's resources.

### CODE REVIEW

**Review by iamniting — APPROVED** [review]

File: deploy/bundle/manifests/volumereplication.crd.yaml [hunk 1]
```diff
@@ -0,0 +1,165 @@
+---
+apiVersion: apiextensions.k8s.io/v1
```
Comment (Madhu-1) [inline deploy/bundle/manifests/volumereplication.crd.yaml:2]: is this a duplicate copy of the CRD? if there are any changes in the CRD in the volume replication operator. Do the same changes need to be done here also?

Comment (umangachapagain) [reply deploy/bundle/manifests/volumereplication.crd.yaml:2]: No. Changes go to Rook. We pull it from Rook image.

Comment (b-ranto) [reply deploy/bundle/manifests/volumereplication.crd.yaml:2]: Rook pulls the CRDs from the volume-replication github repo (and so do we DS) so there is no need to update them in the rook repo.

**Review by b-ranto — APPROVED** [review]

**Comment by umangachapagain** [comment]
These resources will be owned by Rook. No change required.

**Pull request closed without merging**